packages/kbn-test/jest-preset.js

```diff
@@ -76,7 +76,7 @@ module.exports = {
 
   // A list of paths to snapshot serializer modules Jest should use for snapshot testing
   snapshotSerializers: [
-    '<rootDir>/src/plugins/kibana_react/public/util/test_helpers/react_mount_serializer.ts',
```

Comment:
@dmlemeshko do you know anything about the snapshot serializers for jest?
@pheyos what about you Robert, any idea what this is.   
I'll start googling now too